lib/link.js

```diff
@@ -50,6 +50,14 @@ function link (args, cb) {
   linkPkg(npm.prefix, cb)
 }
 
+function parentFolder (id, folder) {
```

Comment:
Maybe I'm overly paranoid, but I'd also like a check that guarantees there's a `/` in the name before adding a second parent directory into the mix. Is that too paranoid?